tests/test_app/templates/test_include.twig

```diff
@@ -1 +1,2 @@
-{{ include('simple.twig') }}
\ No newline at end of file
+{% extends 'TestTwigView.base' %}
```

Comment:
How would extends work from with in an element or cell? Would it search the `App.paths.templates` or would it be limited to the directory for cells/elements. It would be good to include how it works in the readme and/or tests.

Reply:
Updated to be specific about template conventions.